src/components/CreateUser.vue

```
@@ -0,0 +1,171 @@
+  }
+
+  try {
+    isSubmitting.value = true;
+
+    const createdUser = await userService.createUser(user.value, password.value);
+
+    if (createdUser) {
+      successMessage.value = `User ${createdUser.email} created successfully!`;
+      resetForm();
+    }
+  } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)